### ISSUE TITLE
Can't launch Final Fantasy IX 

### ISSUE THREAD

**Issue by firath** [OpenEmu/OpenEmu#3753]
Hello

 I tried to play FFIX, and even though i followed all the step, it didn't work (the game seems to not be recognized by OpenEmu). What i did : 
- Put all the 4 .cue and .bin files on the same folder (French version), with the same corresponding names (CD1, CD2, CD3, CD4)
- Put all 4 .sbi files (French version) on the same folder, with the same corresponding names (CD1, CD2, CD3, CD4)
- Create the M3U file (plain text) where i put the name of all the files in the folder

When i tried to put the m3u file on OpenEmu, here what the console said : 
- Claim 839E62D5-FE5B-4C46-B8C3-CCC670D92F89 invoked in client
- Claim 839E62D5-FE5B-4C46-B8C3-CCC670D92F89 granted in client
- Read options: 0 -- URL: <private> -- purposeID: C02EB437-4C07-4950-81A8-2C80C069820A -- claimID: 839E62D5-FE5B-4C46-B8C3-CCC670D92F89

You also find attached a screenshot of my folder... I'm kindda lost. Would you please help me? 
<img width="1107" alt="capture d ecran 2018-09-22 a 16 07 24" src="https://user-images.githubusercontent.com/42660548/45918067-810d6f00-be81-11e8-8569-1c54f902fe64.png">

Thank you

OS : macOS High Sierra
Processor : 2,7 GHz Intel Core i5
RAM : 8 Go 1867 MHz DDR3

**Comment by clobber** [comment]
Read the guide carefully: https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-CD-based-games#q-i-have-a-multi-disc-game

Sorry, I can't help you beyond that.

**Comment by firath** [comment]
I did... and i'm sorry but i just can't see what i'm doing wrong. 
"Drag and drop the .sbi to import it alongside your game. IMPORTANT — The .sbi file must be named exactly as the base filename of your cue sheet including case e.g."
I did this exactly. I mean, i know we must read the user guide and this is what i did, thoroughly. I tried to be as precise as possible in my explanation so i can have an answer on the possible mistake that i made...
